src/App.scss

```diff
@@ -4,48 +4,98 @@ $breakpoint-tablet: 768px;
 $breakpoint-laptop: 992px;
 $breakpoint-desktop: 1200px;
 
-#root {
-  max-width: 1280px;
-  margin: 0 auto;
-  padding: 2rem;
-  text-align: center;
+.error-img-wrapper {
+  display: flex;
+  align-items: center;
+  justify-content: center;
 }
 
-.logo {
-  height: 6em;
-  padding: 1.5em;
-  will-change: filter;
+.error-img {
+  max-width: 400px;
+  max-height: 400px;
 }
-.logo:hover {
-  filter: drop-shadow(0 0 2em #646cffaa);
+
+// Loader
+
+.loading-bar-wrapper {
+  display: flex;
+  align-items: center;
+  justify-content: center;
+  -webkit-box-sizing: border-box;
+       -moz-box-sizing: border-box;
```

Comment:
NIT: Vite will auto prefix this values so you can remove this. Just having `box-sizing` is enough.

Reply:
Removed.